swap_meet/vendor.py

```diff
@@ -1,2 +1,85 @@
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory=None):
+        self.inventory = inventory or []
+    
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+    
+    def remove(self, item):
+        if item not in self.inventory:
+            return False
+        
+        self.inventory.remove(item)
+        return item
+    
+    def get_by_id(self, num=None):
+        for item in self.inventory:
+            if item.id == num:
+                return item
+        return None
+
+    def swap_items(self, other_vendor, my_item, their_item):
+        if my_item not in self.inventory or their_item not in other_vendor.inventory:
+            return False
+
+        other_vendor.add(my_item)
+        self.add(their_item)
+        other_vendor.remove(their_item)
+        self.remove(my_item)
```

Comment:
My one challenge here would to be to see if you can combine these four lines into just two! Otherwise it looks great!